src/RiakClient/Models/Index/RiakStreamedIndexResult.cs

```diff
@@ -24,17 +24,21 @@ namespace RiakClient.Models.Index
     using Extensions;
     using Messages;
 
+    /// <summary>
+    /// Represents a result to a streaming index query.
+    /// </summary>
     public class RiakStreamedIndexResult : IRiakIndexResult
     {
         private readonly IEnumerable<RiakResult<RpbIndexResp>> responseReader;
         private readonly bool includeTerms;
 
-        public RiakStreamedIndexResult(bool includeTerms, IEnumerable<RiakResult<RpbIndexResp>> responseReader)
+        internal RiakStreamedIndexResult(bool includeTerms, IEnumerable<RiakResult<RpbIndexResp>> responseReader)
```

Comment:
It shouldn't need to be public, the only thing that constructs the results is the RiakClient.cs class, the rest of the interface is public for consumption. (Unless I'm missing something).